src/reducers/currentFilm.js

```diff
@@ -0,0 +1,10 @@
+import { GET_CURRENT_FILM } from '../actions/constans';
+
+const current = (state = {}, action) => (
+  (action.type === GET_CURRENT_FILM) ? {
```

Comment:
It is not extensible. If we want to add more actions it will be pretty complicated. let's use switch instead